Sources/XCTest/Private/ArgumentParser.swift

```diff
@@ -17,10 +17,10 @@ internal struct ArgumentParser {
 
     /// The basic operations that can be performed by an XCTest runner executable
     enum ExecutionMode {
-        /// Run a test or test suite, printing results to stdout and exiting with
-        /// a non-0 return code if any tests failed. The name of a test or class
-        /// may be provided to only run a subset of test cases.
-        case run(selectedTestName: String?)
+        /// Run tests or test cases, printing results to stdout and exiting with
+        /// a non-0 return code if any tests failed. The names of tests or test cases
+        /// may be provided to only run a subset of them.
+        case run(selectedTestNames: [String]?)
```

Comment:
I guess we should prefer to use a `Set` for this for efficiency when filtering during execution?

Reply:
We will iterate over this array in `TestFiltering.selectedTestFilter` anyway for mapping into `SelectedTest`s, but using a Set in the filter itself is indeed faster. Fixed.